lib/ui/screens/about.dart

```diff
@@ -0,0 +1,271 @@
+import 'package:cobble/ui/router/cobble_screen.dart';
+import 'package:cobble/ui/theme/with_cobble_theme.dart';
+import 'package:flutter/material.dart';
+import 'package:url_launcher/url_launcher.dart';
+import 'package:package_info/package_info.dart';
+import 'package:flutter_svg_provider/flutter_svg_provider.dart';
+import 'dart:io' show Platform;
+import 'package:intl/intl.dart';
+
+class About extends StatefulWidget implements CobbleScreen {
+  @override
+  State<StatefulWidget> createState() => _AboutState();
+}
+
+class _AboutState extends State<About> {
+  static const discordURL = "https://discord.gg/aRUAYFN";
+  static const helpCenterURL = "https://help.rebble.io/";
+  static const redditURL = "https://reddit.com/r/pebble";
+  static const emailURL = "mailto:support@rebble.io";
+  static const twitterURL = "https://twitter.com/pebble_dev";
+  static const sourceURL = "https://github.com/pebble-dev/mobile-app";
+  String _appVersion = '';
+  String _appName = '';
+  String os = toBeginningOfSentenceCase(Platform.operatingSystem) ?? '';
+
+  @override
+  void initState() {
+    super.initState();
+    // Apparently there is no pretty print platform name, so we just set the exceptions here
+    if (Platform.isIOS)
+      os = "iOS";
+    else if (Platform.isMacOS)
+      os = "macOS";
+    initPlatformState();
+  }
+
+  initPlatformState() async {
+    PackageInfo packageInfo = await PackageInfo.fromPlatform();
+    String appName = packageInfo.appName;
+    String appVersion = packageInfo.version;
+
+    if (!mounted) return;
+
+    setState(() {
+      _appName = appName;
+      _appVersion = appVersion;
+    });
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return CobbleScaffold.tab(
+      title: " ", // We want an empty appbar
+      bottom: PreferredSize(
+        preferredSize: const Size.fromHeight(156.0),
+        child: Container(
+            height: 156.0,
+            child: Padding(
+              padding: EdgeInsets.fromLTRB(16.0, 14.0, 16.0, 9.0),
+              child: Column(children: <Widget>[
+                Row(children: <Widget>[
+                  SizedBox(
+                    width: 64.0,
+                    height: 64.0,
+                    child: Image(image: Svg('images/app_icon.svg')),
+                  ),
+                  SizedBox(width: 20.0),
+                  Column(
+                      crossAxisAlignment: CrossAxisAlignment.start,
+                      children: <Widget>[
+                        Text(
+                          _appName,
+                          style: Theme.of(context).textTheme.headline5,
+                        ),
+                        SizedBox(height: 4.0),
+                        Text(
+                          tr.aboutPage.versionString(
+                              version: _appVersion, platform: os),
+                          style: Theme.of(context).textTheme.bodyText2,
+                        ),
+                      ]),
```

Comment:
Flutter's recommendation is to use trailing comma in order to get nicer formatted code:
https://flutter.dev/docs/development/tools/formatting#using-trailing-commas
I suggest putting comma after every square bracket or parenthesis.